toolset/setup/linux/webservers/resin-java8.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+RVER=4.0.41
+
+RETCODE=$(fw_exists ${IROOT}/resin-java8-$RVER.installed)
+[ ! "$RETCODE" == 0 ] || { return 0; }
+
+fw_depends java8
+sudo cp -r $JAVA_HOME/include $JAVA_HOME/jre/bin/
```

Comment:
Oh god, no. Let's not do this anymore... this version thing is officially untenable. I see no difference between this file you have added and [this file](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/webservers/resin.sh) aside from depending on Java8. To me, that reads "resin will work with java8 or java7"

Question - if you were to leave resin alone and still have `source $IROOT/java8.installed` in the setup file, will it run?